contrib/babelfishpg_tsql/sql/information_schema_tsql.sql

```diff
@@ -716,4 +716,33 @@ CREATE OR REPLACE VIEW information_schema_tsql.routines AS
 
 GRANT SELECT ON information_schema_tsql.routines TO PUBLIC;
 
+/*
+ * SCHEMATA view
+ */
+CREATE OR REPLACE VIEW information_schema_tsql.schemata AS
```

Comment:
for testing, this can have impact on tooling and should be validated. Also, we should compare with SQL server to make sure that view has reasonable perf for large number of objects.